src/bootstrap.ts

```diff
@@ -49,6 +49,32 @@ export const bootstrap = async () => {
 
   // Health check
   app.get('/healthcheck', (_, res) => res.sendStatus(200))
+  app.get('/hc', function (_, res) {
+    res.redirect('/healthcheck')
```

Comment:
I would not use redirect. It will send status 302 and you have to setup follow redirects on the health check client. Just create `(_, res) => res.sendStatus(200)` function and use it in `/hc` and `/healhtcheck` endpoints

Reply:
ok, fixed